example/infra/caches.py

```diff
@@ -0,0 +1,55 @@
+from pathlib import Path
+from shutil import copy, copytree
+from typing import Optional
+from blockwork.build.caching import Cache
+from blockwork.context import Context
+
+class BasicFileCache(Cache):
```

Comment:
Wondering why you don't use the state handling stuff here?

https://github.com/blockwork-eda/blockwork/blob/main/blockwork/state.py

Reply:
I didn't think of it, but also I think it's useful for the example file cache to be as simple as possible so users can easily see what they would need to do to implement another version